e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -182,4 +182,27 @@ describe('Message', () => {
         // * Focus to remain in the RHS text box
         cy.get('#reply_textbox').should('be.focused');
     });
+
+    it('MM-T336 Image thumbnail - expanded RHS', () => {
+        const filename = 'huge-image.jpg';
+
+        // # Post an image in center channel
+        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
+        cy.contains(filename).should('be.visible');
+        cy.postMessage('{enter}');
+
+        // # Click reply arrow to open the reply thread in RHS
+        cy.clickPostCommentIcon();
+
+        cy.get('#rhsContainer').within(() => {
+            // # Observe image thumbnail displays the same
+            cy.get('img[src*="/preview"]').should('be.visible');
+
+            // # In the RHS, click the expand arrows to expand the RHS
+            cy.get('button[aria-label="Expand"]').click();
```

Comment:
The expand button does not have any `id` or `data-cy` etc attributes. Is there a ticket to add missing `id` such as this one?

Reply:
No separate ticket. We normally add it as we write E2E. Also, we are not using `data-cy`.
However, if it can be [queried via testing-library](https://testing-library.com/docs/dom-testing-library/api-queries), the would be better. So in this case, you may use `cy.findByLabelText('Expand').click();`